examples/tokenvm/tests/e2e/e2e_test.go

```diff
@@ -1361,12 +1266,12 @@ var _ = ginkgo.Describe("[Test]", func() {
 		)
 		gomega.Expect(err).To(gomega.BeNil())
 
-		// Wait some time before broadcasting new txs so that state sync can finsih
+		// Wait some time before broadcasting new txs so that state sync can finish
 		// without updating roots
 		//
 		// TODO: find a way to communicate this explicitly without sleeping
 		time.Sleep(30 * time.Second) // assume state sync takes less than this to complete
```

Comment:
I think this can be removed. I believe this is a remnant of not setting the health check properly in the past.